change/@fluentui-react-accordion-31bad2ca-70dc-4308-a81d-d098d01134cf.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32304&commitId=2aff110424068cae1abee0c77cd47fe5dc78c0f7&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|8| Changed|

</details>


  <details><summary><b>Card Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.Appearance High Contrast.chromium.png|5345| Changed|
|Card Converged.Appearance Dark Mode.chromium.png|366| Changed|
|Card Converged.Appearance RTL.chromium.png|276| Changed|
|Card Converged.appearance.chromium.png|276| Changed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.Appearance Interactive Outline Dark Mode.chromium.png|242| Changed|
|Card Converged - Interactive.appearance interactive - Outline.chromium.png|183| Changed|
|Card Converged - Interactive.Appearance Interactive Outline High Contrast.chromium.png|1014| Changed|
|Card Converged - Interactive.Appearance Interactive Filled High Contrast.chromium.png|1014| Changed|
|Card Converged - Interactive.Appearance Interactive Subtle High Contrast.chromium.png|1014| Changed|
|Card Converged - Interactive.Appearance Interactive Filled Alternative High Contrast.chromium.png|1014| Changed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.Appearance Selectable Filled Alternative High Contrast.chromium.png|1014| Changed|
|Card Converged - Selectable.Appearance Selectable Subtle High Contrast.chromium.png|1014| Changed|
|Card Converged - Selectable.Appearance Selectable Outline RTL.chromium.png|183| Changed|
|Card Converged - Selectable.Appearance Selectable Outline High Contrast.chromium.png|1014| Changed|
|Card Converged - Selectable.Appearance Selectable Outline Dark Mode.chromium.png|242| Changed|
|Card Converged - Selectable.appearance selectable - Outline.chromium.png|183| Changed|
|Card Converged - Selectable.Appearance Selectable Filled High Contrast.chromium.png|1014| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|2219| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>